README.md

```diff
@@ -702,22 +702,23 @@ _Create an object containing three enhanced action-creators to simplify handling
 ```ts
 createAsyncAction(
   requestType, successType, failureType, cancelType?
-)<TRequestPayload, TSuccessPayload, TFailurePayload, TCancelPayload?>()
+)<TRequestPayload, TSuccessPayload, TFailurePayload, TCancelPayload?,
+  TRequestMeta?,   TSuccessMeta?,   TFailureMeta?,   TCancelMeta?>()
 ```
 
 ##### `AsyncActionCreator`
 
 ```ts
 type AsyncActionCreator<
-  [TRequestType, TRequestPayload],
-  [TSuccessType, TSuccessPayload],
-  [TFailureType, TFailurePayload],
-  [TCancelType, TCancelPayload]?
+  [TRequestType, TRequestPayload, TRequestMeta?],
```

Comment:
This 3rd parameter is definitely a no-go because it's not compatible with new v5 changes